platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/GenesisStateBuilder.java

```diff
@@ -68,11 +64,18 @@ public static ReservedSignedState buildGenesisState(
             @NonNull final MerkleRoot stateRoot) {
 
         final BasicConfig basicConfig = platformContext.getConfiguration().getConfigData(BasicConfig.class);
-        stateRoot.setPlatformState(buildGenesisPlatformState(addressBook, appVersion));
+        final PlatformStateAccessor platformState;
```

Comment:
Where is this platform state object used? It's created below, then initialized in `initGenesisPlatformState()`, and then ...?